xarray/tests/test_plot.py

```diff
@@ -91,14 +91,44 @@ def setUp(self):
     def test1d(self):
         self.darray[:, 0, 0].plot()
 
-        with raises_regex(ValueError, 'dimension'):
+        with raises_regex(ValueError, 'None'):
             self.darray[:, 0, 0].plot(x='dim_1')
 
+    def test_1d_x_y_kw(self):
+        z = np.arange(10)
+        da = DataArray(np.cos(z), dims=['z'], coords=[z], name='f')
+
+        xy = [[None, None],
+              [None, 'z'],
+              ['z', None]]
+
+        f, ax = plt.subplots(2, 4)
+
+        for aa, (x, y) in enumerate(xy):
+            da.plot(x=x, y=y, ax=ax.flat[aa])
+            ax.flat[aa].set_title('x=' + str(x) + ' | ' + 'y=' + str(y))
```

Comment:
Is there any point to this line?

Reply:
No. I copied the test over from my debugging plots. I'll remove it (I assume you mean the `set_title` bit).